### ISSUE TITLE
Project for demonstrating activation functions.

### ISSUE THREAD

**Issue by hackintoshrao** [Kredoai/Deep-Learning-Training#5]
Need project with a neural network with at least two layers and using various activation functions. The idea is to compare the validation accuracy of various activation functions.
